acceptance/tests/facts/windows_os.rb

```diff
@@ -0,0 +1,26 @@
+test_name "Test facter os reports correct os.windows.system32" do
+
+  confine :to, :platform => /windows/
+
+  agents.each do |agent|
```

Comment:
Each test should leave the machine in the same state that he found it. For this we use `teardown` to make sure that all the changes to the system are reverted.

https://github.com/puppetlabs/beaker/blob/574b6cbf3ac59dc1106d2d89918f80ad8afb751a/lib/beaker/dsl.rb#L27

Reply:
teardown in place